public/File_Zipper.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+
+    <link rel="stylesheet" href='../assets/css/File_Zipper.css'>
+    <script src='../assets/css/File_Zipper.css'></script>
+    <script src="../assets/js/File_Zipper_js.js" type="module"></script>
+    <script src="../assets/js/File_Zipper_js.js" type="module"></script>
```

Comment:
Repeated lines. Also, the src is wrong. It has to be 
```
<script src="../assets/js/File_Zipper_js/script.js" type="module"></script>
```

Reply:
done